src/main/java/slimeknights/tconstruct/library/recipe/worktable/ModifierSetWorktableRecipe.java

```diff
@@ -47,7 +47,7 @@ public class ModifierSetWorktableRecipe extends AbstractWorktableRecipe {
   /** Loader instance */
   public static final RecordLoadable<ModifierSetWorktableRecipe> LOADER = RecordLoadable.create(
     ContextKey.ID.requiredField(),
-    Loadables.RESOURCE_LOCATION.requiredField("data_key", r -> r.dataKey),
+    Loadables.RESOURCE_LOCATION.requiredField("data_key", r -> r.key),
```

Comment:
Why did this file change? Overzealous find and replace?

Reply:
I believe so.